apier/v1/suppliers_it_test.go

```diff
@@ -673,6 +674,103 @@ func testV1SplSGetQOSSuppliers3(t *testing.T) {
 	}
 }
 
+func testV1SplSGetQOSSuppliersFiltred(t *testing.T) {
+	ev := &engine.ArgsGetSuppliers{
+		CGREvent: utils.CGREvent{
+			Tenant: "cgrates.org",
+			ID:     "testV1SplSGetQOSSuppliers",
+			Event: map[string]interface{}{
+				"DistincMatch": "*qos_filtred",
+			},
+		},
+	}
+	eSpls := engine.SortedSuppliers{
+		ProfileID: "SPL_QOS_FILTRED",
+		Sorting:   utils.MetaQOS,
+		SortedSuppliers: []*engine.SortedSupplier{
+			&engine.SortedSupplier{
+				SupplierID: "supplier1",
+				SortingData: map[string]interface{}{
+					"*acd:Stat_1":   11.0,
+					"*acd:Stat_1_1": 11.0,
```

Comment:
would be interesting to see a different *acd here so we can see them properly using worst stat